java/bogdana_maksiuta/src/com/zoolatech/lecture3/tasks/_1/Drum.java

```diff
@@ -0,0 +1,16 @@
+package com.zoolatech.lecture3.tasks._1;
+
+public class Drum extends MusicalInstrument {
```

Comment:
Do we really need to create a class for each instrument? Do they provide some additional functionality?

Reply:
no, we don't need it. fixed